src/RoleObjects/CompositeData.test.js

```diff
@@ -0,0 +1,54 @@
+import * as createjs from 'createjs-module';
+import AccessibilityModule from '../index';
+import { parentEl, stage, container } from '../__jestSharedSetup';
+
+describe('CompositeData', () => {
+  describe('register role', () => {
+    let cjsComposite;
+    let compositeEl;
+
+    beforeEach(() => {
+      cjsComposite = new createjs.Shape();
+      AccessibilityModule.register({
+        displayObject: cjsComposite,
+        parent: container,
+        role: AccessibilityModule.ROLES.APPLICATION,
+      });
+
+      stage.accessibilityTranslator.update();
+      compositeEl = parentEl.querySelector(`#${cjsComposite.accessible.domId}`);
+    });
+
+    describe('rendering', () => {
+      it('creates composite application element', () => {
+        expect(compositeEl).not.toBeUndefined();
+      });
+    });
+
+    describe('accessible options getters and setters', () => {
```

Comment:
- when we've used this `describe` in other tests, `accessible` was referring to attributes starting with `aria-`; apologies if this was confusing since the API uses the `accessible` property for a different purpose

Reply:
Okay! Got it.
I can name this `describe` as "getters and setters methods" to keep things simple!